README.md

```diff
@@ -4,31 +4,49 @@
 </a>
 
 [![Build status](https://img.shields.io/travis/SURFnet/sp-dashboard.svg)](https://travis-ci.org/SURFnet/sp-dashboard)
+[![License](https://img.shields.io/github/license/SURFnet/sp-dashboard.svg)](https://github.com/SURFnet/sp-dashboard/blob/master/LICENSE.txt)
 
 # Service Provider Dashboard
 
-Dashboard for [SURFconext](https://www.surf.nl/diensten-en-producten/surfconext/index.html) Service Providers.
+The Service Provider Dashboard is a dashboard application where
+[SURFconext](https://www.surf.nl/diensten-en-producten/surfconext/index.html) Service Providers can register and manage
+their services.
 
 ## Prerequisites
 
+- [PHP](https://secure.php.net/manual/en/install.php) (5.6 or higher)
 - [Composer](https://getcomposer.org/doc/00-intro.md)
+- [Apache Ant](https://ant.apache.org/manual/install.html)
 - [Ansible](https://docs.ansible.com/ansible/intro_installation.html)
 - [Vagrant](https://www.vagrantup.com/docs/installation/)
   - Optional, but recommended: [Hostsupdater plugin](https://github.com/cogitatio/vagrant-hostsupdater)
 
+The Ansible playbook for SP Dashboard depends on some roles from
+[OpenConext-deploy](https://github.com/OpenConext/OpenConext-deploy), so in order to provision the Vagrant box you need
+to have that repository checked out in a directory called `OpenConext-deploy` in the parent directory of where this
+project lives.
+
 ## Getting started
 
-Install Composer dependencies using:
+First, run `composer install`. This will install all PHP dependencies, including the development dependencies.
+
+In order to start the development environment, run `vagrant up`. This will build and start the virtual machine that is
+used in development to run the application. When started for the first time, the Vagrant box will be provisioned using
+Ansible.
+
+Note: if you don't use the Vagrant Hostsupdater plugin, you have to manually add
+`192.168.33.19  dev.support.surfconext.nl` to your hosts file so that requests will be routed to the virtual machine.
```

Comment:
While checking my `hosts` file, I noticed this IP is used for another SurfConext project. `serviceregistry.dev.support.surfconext.nl`. It might be a good idea to use another IP for this project. Not really an issue with the documentation, just saying.